elastic/kibana
# [Guided onboarding] Security tour

## Summary
Addresses https://github.com/elastic/cloud/issues/102847

This PR adds an onboarding tour for the Security solution. A clickable design prototype is available under this [link](https://ela.st/onboarding-prototype) (select v 8.4). Related to a similar onboarding tour for Observability (https://github.com/elastic/kibana/pull/133909).
Tests will be added as a follow up PR (tracked in https://github.com/elastic/cloud/issues/104482).

### Screenshots 
<img width="1469" alt="Screenshot 2022-07-20 at 14 28 36" src="https://user-images.githubusercontent.com/6585477/179993001-b0af9011-89fb-4867-88bf-d71729f1ca73.png">


<details>

<img width="1469" alt="Screenshot 2022-07-20 at 14 28 42" src="https://user-images.githubusercontent.com/6585477/179993297-4281ed7f-2557-41a1-8a7d-a4dc2e0ed247.png">

<img width="1469" alt="Screenshot 2022-07-20 at 14 28 50" src="https://user-images.githubusercontent.com/6585477/179993541-910e87fe-5fa4-48a6-8341-d011b489b47c.png">

<img width="1469" alt="Screenshot 2022-07-20 at 14 29 03" src="https://user-images.githubusercontent.com/6585477/179993670-f91251f5-4261-4ec9-812e-2731eb90b3cc.png">

<img width="1469" alt="Screenshot 2022-07-20 at 14 29 17" src="https://user-images.githubusercontent.com/6585477/179993847-c8551a74-61b4-4a5e-9e80-1f0fad44c97e.png">


</details>

### Screencast

https://user-images.githubusercontent.com/6585477/179992524-b90a1998-0111-48be-bcd7-cac192f38cdc.mov





### How to test
1. Navigate to Stack Management -> Advanced Settings and enable the new Security navigation (set `securitySolution:enableGroupedNav` to ON)
2. Navigate to /home#/getting_started and select Security
3. Check that the tour is correctly displayed: 
  - The tour can be skipped.
  - The tour can be completed.
  - When the tour is displayed, the side navigation can't be collapsed. 
  - The tour is not displayed on small screens (xs and s breakpoints).
  - The tour is displayed even if `hideAnnoucements` setting is ON, because those onboarding tours are activated by the user action and not displayed by default. 


### Checklist
